poster/src/post_with_retries.ts

```diff
@@ -55,6 +59,14 @@ async function postWithRetries(transaction: TransactionConfig, signerKey: string
       };
     }
 
+    if (maybeIsOutOfGas(e)) {
+      console.log('Transaction is possibly out of gas, increasing gas limit');
+      transaction = {
+        ...transaction,
+        gas: Math.floor(Number(transaction.gas) * GAS_ADJUSTMENT)
```

Comment:
Maybe we should always apply at least one level of adjustment?

Reply:
@coburncoburn @hayesgm any thoughts here? I can increase the gas limit on each retry to simplify logic here